applications/portal/backend/api/views/rest/population.py

```diff
@@ -1,25 +1,46 @@
 import logging
 
-# import numpy as np
 from django.http import Http404, HttpResponse, FileResponse
-
 from dry_rest_permissions.generics import DRYPermissions
 from rest_framework import status, viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
+from rest_framework.schemas.openapi import AutoSchema
 
 from api.constants import PopulationPersistentFiles
-from api.helpers.atlas import get_bg_atlas
-from api.helpers.exceptions import DensityMapIncorrectSubdivisionError
 from api.models import Experiment, Population
 from api.serializers import PopulationSerializer
 from api.services.population_service import get_cells
 from api.utils import send_file
-from api.validators.density_map import validate_subdivision
 
 log = logging.getLogger("__name__")
 
 
+class CustomPopulationSchema(AutoSchema):
```

Comment:
The automatic generation of the openapi specifation was producing an incorrect return type for the retrieval of residential populations